.setup/bin/sample_courses/models/__init__.py

```diff
@@ -238,7 +230,7 @@ def generate_random_users(total, real_users):
             user_id = user_id.lower()
             anon_id = generate_random_user_id(15)
             # create a binary string for the numeric ID
-            numeric_id = "{0:09b}".format(i)
+            numeric_id = f"{i:09b}"
```

Comment:
I'm pretty sure this is the correct way to convert .format() to f-strings, but otherwise everything else is converted to f-strings